apps/feedahelp-cms/schemas/ingredient.ts

```diff
@@ -0,0 +1,47 @@
+import {LemonIcon} from '@sanity/icons'
+import {defineField, defineType} from 'sanity'
+
+export default defineType({
+  name: 'ingredient',
+  title: 'Ingredient',
+  type: 'document',
+  icon: LemonIcon,
+  fields: [
+    defineField({
+      name: 'ingredientName',
+      title: 'Ingredient Name',
+      type: 'string',
+      validation: (Rule) => Rule.required(),
+    }),
+    defineField({
+      name: 'ingredientNameBangla',
+      title: 'Ingredient Name Bangla',
+      type: 'string',
+      validation: (Rule) => Rule.required(),
+    }),
+    defineField({
+        name: 'ingredientImage',
+        title: 'Ingredient Image',
+        type: 'image',
+        validation: (Rule) => Rule.required(),
+    }),
```

Comment:
We need another `imgURL`
https://drive.google.com/file/d/1PgJJFhdHbzBtWIzUxO3l2Yk94x3arKGs/view?usp=drive_link